### PR TITLE
ci: Configure the GitHub Actions

### PR DESCRIPTION
**Pull request by taorepoara**
## Checklist
- [x] I didn't over-scope my PR
- [x] My PR title matches the [commit convention](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I did not include breaking changes
- [ ] I included unit tests that cover my changes
- [ ] I added/updated the documentation about my changes
- [x] I made my own code-review before requesting one

## Description of the changes
Configure the GitHub Actions to build, test and release the project.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/task.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+Describe the task and scope...
+
+## Technical recommandation
+Any technical recommandation or specific warning about this task (if needed)
+- Security issue to be warned of 
+- Complexity issue
+
+## Ecological concerns
+What can be done to improve the ecological impact of this task ?
+- Can we minimize the data send to the network ?
```
Comment (jonas-martinez) [inline .github/ISSUE_TEMPLATE/task.md:21]:
```suggestion
- Can we minimize the data sent to the network ? 
```

**Review by jonas-martinez — APPROVED** [review]
Looks good to me !
Waiting for a review from @shiipou just to be sure.

File: .github/workflows/build_ci.yml [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v2
+      with:
+        node-version: "16"
+
+    - name: Setup node deps
+      shell: bash
+      run: npm i conventional-changelog-conventionalcommits@4 @semantic-release/git semantic-release-export-data -D
+
```
Comment (shiipou) [inline .github/workflows/build_ci.yml:204]:
```suggestion
```

File: .github/workflows/build_ci.yml [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+    - id: release
+      name: Release
+      shell: bash
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: npx semantic-release@18
```
Comment (shiipou) [inline .github/workflows/build_ci.yml:210]: Maybe use it with only one line : 
```suggestion
      run: npx -p conventional-changelog-conventionalcommits@5 -p @semantic-release/git -p semantic-release-export-data -p semantic-release@19 semantic-release
```

File: .github/workflows/build_ci.yml [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+      uses: actions/checkout@v3
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: 16
+
+    - name: Setup node deps
+      shell: bash
+      run: npm i conventional-changelog-conventionalcommits@4 @semantic-release/git semantic-release-export-data -D
```
Comment (shiipou) [inline .github/workflows/build_ci.yml:34]:
```suggestion
```

File: .github/workflows/build_ci.yml [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+
+    - id: get-next-version
+      name: Get next version
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: npx semantic-release --dry-run
```
Comment (shiipou) [inline .github/workflows/build_ci.yml:40]: Idem 
```suggestion
      run: npx -p conventional-changelog-conventionalcommits@5 -p @semantic-release/git -p semantic-release-export-data -p semantic-release@19 semantic-release --dry-run
```

**Review by shiipou — CHANGES_REQUESTED** [review]

**Pull request merged**